src/pkg/export/consume.go

```diff
@@ -28,15 +28,15 @@ func ConsumeExportCollections(
 		folder := filepath.Join(exportLocation, col.BasePath())
 		ictx := clues.Add(ctx, "dir_name", folder)
 
-		for item := range col.Items(ctx) {
+		for item := range col.Items(ictx) {
 			if item.Error != nil {
-				el.AddRecoverable(ictx, clues.Wrap(item.Error, "getting item").WithClues(ctx))
+				el.AddRecoverable(ictx, clues.Wrap(item.Error, "getting item"))
```

Comment:
just checking, but will we lose `dir_name` (added on line 29) if we don't do `WrapWC` here?

Reply:
Standard assumption is that any corso-owned func that takes in a context (`Items(ctx)` in this case) is responsible for calling WithClues(ctx) on any error it generates.  In this case, we'll only lose `dir_name` if `Items()` is misbehaving.